priceloop_api/priceloop/extras/exceptions.py

```diff
@@ -0,0 +1,22 @@
+class ColumnsMismatch(Exception):
+    pass
+
+
+class ColumnTypesMismatch(Exception):
+    pass
+
+
+class ColumnMixedTypes(Exception):
+    pass
+
+
+class TypeNotFound(Exception):
+    pass
+
+
+class NotFoundPlatformError(Exception):
+    pass
```

Comment:
Exception classes should have an Error suffix according to pep 8, see https://peps.python.org/pep-0008/#exception-names